custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt

```diff
@@ -92,11 +91,6 @@ class CustomReaderFragment : CoreReaderFragment() {
     }
   }
 
-  override fun onBackPressed(activity: AppCompatActivity): FragmentActivityExtensions.Super {
-    requireActivity().finish()
-    return super.onBackPressed(activity)
-  }
-
   private fun openSearchItem(item: SearchItemToOpen) {
```

Comment:
Why this method is overriden?

Reply:
talking about opensearchItem() and the question is why it need to be doffetent in a custom app?

Reply:
@kelson42 Thanks for highlighting this, We are observing the `seachedItem` in both modules(app, custom) and this method for opening the searched item, but we have written this method twice in both reader fragments so it would be better to move this code `CoreReaderFragment`.

Reply:
Done.